src/booking/containers/booking-date/booking-date.component.ts

```diff
@@ -1,25 +1,84 @@
 import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
-import { BookingNavigationEvent, NewBooking } from '../../models';
+import {  BookingNavigationEvent, NewBooking} from '../../models';
+import {
+  FormBuilder,
+  FormControl,
+  FormGroup,
+  Validators,
+} from '@angular/forms';
+import {
+  TimeOption,
+  BookingTimePageText,
+} from '../../models/booking-date.enum';
+import { ErrorMessage } from '../../../app/models';
+import { GovUiConfigModel } from '@hmcts/rpx-xui-common-lib/lib/gov-ui/models';
 
 @Component({
   selector: 'exui-booking-date',
   templateUrl: './booking-date.component.html',
-  styleUrls: ['./booking-date.component.scss']
+  styleUrls: ['./booking-date.component.scss'],
```

Comment:
Why the comma?

Reply:
Hi @paulhowes-HMCTS - resolving on your behalf since @tarikdogan-hmcts has removed `styleUrls` entirely (the SCSS file is no longer required).